pyproject.toml

```
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "stumpy"
+version = "1.12.0"
+requires-python = ">=3.8"
+authors = [
+    {name = "Sean M. Law", email = "seanmylaw@gmail.com"}
```

```suggestion
    {name = "Sean M. Law"}, 
    {email = "seanmylaw@gmail.com"},
```

Otherwise, this will not be shown properly in the output returned by `pip show stumpy`

---

Btw, `pip show stumpy` does not show the "homepage". 

However, `pip show stumpy --verbose` shows an extra homepage field that contains the value used in the toml file. So, I think this should be okay (?)

---

Please ignore (see [this comment](https://github.com/TDAmeritrade/stumpy/pull/955#issuecomment-1939111838))